examples/images_contours_and_fields/interpolation_none_vs_nearest.py

```diff
@@ -1,62 +0,0 @@
-"""
```

Comment:
It does not have to be an example but I would be very sorry to see that figure go. I have on several occasions benefitted from having this figure to explain the difference.  A graphical explanation is so much more powerfull than text only. The [interpolation_methods.py](http://matplotlib.org/examples/images_contours_and_fields/interpolation_methods.html) example does nothing to explain this as `None` `none` and `nearest` are rendered identically